__init__.py

```diff
@@ -47,6 +52,13 @@ class SystemCommand(Enum):
 
 
 class DeviceControlCenterSkill(NeonSkill):
+    user_config_path = expanduser("~/.config/neon/neon.yaml")
+
+    def initialize(self):
+        self.bus.on("mycroft.ready", self._speak_restart_dialog)
```

Comment:
`self.add_event` may be preferable if you want the generic error handler to speak a generic error if something goes wrong here

Reply:
Not necessarily...this is really a fallback to getting the new personality to speak to the user, and is now covered by the skill initialization itself. I'd rather it not speak an error without any context, honestly

Reply:
PS: self.add_event should still be used to ensure proper removal of handlers on shutdown, it also has a flag to to speak the error or not

Reply:
<img width="427" alt="image" src="https://github.com/NeonGeckoCom/skill-device_controls/assets/30268971/db40db27-5dbc-4898-bbe3-389ffd0beb06">